frame/balances/src/tests_reentrancy.rs

```diff
@@ -197,15 +204,22 @@ fn transfer_dust_removal_tst2_should_work() {
 		// Verify the events
 		assert_eq!(System::events().len(), 10);
 
-		System::assert_has_event(Event::Balances(crate::Event::Transfer(2, 1, 450)));
-		System::assert_has_event(Event::Balances(crate::Event::DustLost(2, 50)));
-		System::assert_has_event(Event::Balances(crate::Event::Deposit(1, 50)));
+		System::assert_has_event(Event::Balances(crate::Event::Transfer {
+			from: 2,
+			to: 1,
+			amount: 450,
+		}));
+		System::assert_has_event(Event::Balances(crate::Event::DustLost {
+			account: 2,
+			amount: 50,
+		}));
+		System::assert_has_event(Event::Balances(crate::Event::Deposit { who: 1, amount: 50 }));
 	});
 }
 
 #[test]
 fn repatriating_reserved_balance_dust_removal_should_work() {
-	ExtBuilder::default().existential_deposit(100).build().execute_with(|| {
+	ExtBuilder::default().existential_deposit(200).build().execute_with(|| {
```

Comment:
This is concerning. Behavior should not change with this PR. Please revert this back and try if it works.

Reply:
Yeah, that was an oversight, the main error was from the assertion, I’ll revert this.